app/database.js

```diff
@@ -0,0 +1,54 @@
+/* file: database.js
+ * authors: H. Bouakaz, S. de Vliet, S. de Jong & E. Werkema
+ * date: 13/3/2016
+ * version 1.1
+ *
+ * Description:
+ * Module for loading the database. This file contains all the functions that
+ * update or retrieve information from the database. The dependancy for this
+ * module is sqlite.
+ */
+
+define(['sqlite'], function (sqlite) {
+	// Check if SQL.js has been loaded through AMD
+	var sql;
+	if (typeof sqlite !== 'object') {
+		document.body.style.backgroundColor = 'red';
```

Comment:
True, but this is a temporary way of showing that there was a database error. We will make a user-friendly error page later, see https://trello.com/c/c9M0NULR